web/docs/tql2/operators/read_csv.md

```diff
@@ -21,6 +22,24 @@ values instead of dropping the excess values.
 
 Treat lines beginning with "#" as comments.
 
+### `doubled_quotes_escape = bool (optional)`
+
+Whether to support "triple-escaping". That is, whether to consider two consecutive
+quote characters *inside of a quoted string* as an escape sequence. Enabling this option, all these inputs parts the same:
+
+```
+Header1, Header2
+"This is a single field with, ""quoted"" text",Next field
+"This is a single field with, \"quoted\" text",Next field
+This is a single field with\, "quoted" text,Next field
+```
+```tql
+{
+  Header1: "This is a single field with \"quoted\" text",
+  Header2: "Next field"
+}
+```
```

Comment:
Would we want this to be enabled by default? And if yes, should it be an option at all?